CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## How to contribute to Central Dogma Go project
```

Comment:
How about: Central Dogma Go project -> Go client library for Central Dogma ? :sweat_smile: